pkg/javascript/web/src/payload.rs

```diff
@@ -80,6 +80,26 @@ fn convert_payload(payload: Payload) -> Json {
                 "columns": Json::Array(columns),
             })
         }
+        Payload::ExplainTable(columns) => {
+            let columns = columns
+                .into_iter()
+                .map(|(field, data_type, nullable, key, default, extra)| {
```

Comment:
Currently we don't have any fields to show on `extra`.
Looks like it would be better to remove `extra`.